ros_buildfarm/config/build_file.py

```diff
@@ -37,6 +37,8 @@ class BuildFile(object):
     def __init__(self, name, data):
         self.name = name
 
+        self.custom_rosdep_urls = []
+
```

Comment:
Please move this to the `SourceBuildFile` (above line 95) for now. Once any other build file supports additional configuration we can generalize it then.